gradle.properties

```diff
@@ -5,26 +5,26 @@ pluginName = modulite
 pluginRepositoryUrl = https://github.com/VKCOM/modulite
 
 # SemVer format -> https://semver.org
-pluginVersion = 1.1.7
+pluginVersion = 1.1.8
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 242
-pluginUntilBuild = 242.*
+pluginSinceBuild = 243
+pluginUntilBuild = 243.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = IU
-platformVersion = 2024.2
+platformVersion = 2024.3
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.jetbrains.php:203.4449.22, org.intellij.scala:2023.3.27@EAP
 # PHP Plugin - https://plugins.jetbrains.com/plugin/6610-php/versions
 # YAML Plugin - https://plugins.jetbrains.com/plugin/13126-yaml/versions
-platformPlugins = com.jetbrains.php:242.20224.427, org.jetbrains.plugins.yaml:242.20224.237
+platformPlugins = com.jetbrains.php:243.21565.211, org.jetbrains.plugins.yaml:243.21565.211
```

Comment:
`org.jetbrains.plugins.yaml` is `platformBundledPlugins`.

https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html#ids-of-bundled-plugins